src/NovelRT.Interop/Audio/NrtAudioService.cpp

```diff
@@ -94,7 +149,16 @@ NrtResult Nrt_AudioService_pauseMusic(NrtAudioService service) {
     }
 
     auto serv = reinterpret_cast<Audio::AudioService*>(service);
-    serv->pauseMusic();
+
+    try
+    {
+      serv->pauseMusic();
+    }
+    catch (const Exceptions::NotInitialisedException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?